packages/components/src/components/textarea/textarea.tsx

```diff
@@ -107,10 +105,11 @@ export class Textarea {
   @State() hasFocus: boolean = false;
 
   private focusableElement: HTMLElement;
+  private readonly internalId = generateUniqueId();
 
   componentWillLoad() {
     if (this.inputId == null) {
-      this.inputId = 'input-textarea' + i++;
+      this.inputId = 'input-textarea' + this.internalId;
```

Comment:
I would add a dash after "textarea" for consistency (thanks for adding it in the other components) — the rest is ace 👌

Reply:
Oh yes, thank you 🙂